spec/lucky_web/log_handler_spec.cr

```diff
@@ -43,6 +43,32 @@ describe LuckyWeb::LogHandler do
     log_output.should contain("GET")
     log_output.should contain("Unhandled exception:")
   end
+
+  context "when configured to log timestamps" do
+    it "logs timestamp" do
+      begin
+        LuckyWeb::LogHandler.configure do
+          settings.show_timestamps = true
+        end
+
+        io = IO::Memory.new
+        called = false
+        log_io = IO::Memory.new
+        context = build_context_with_io(io)
+
+        call_log_handler_with(log_io, context) { called = true }
+
+        log_output = log_io.to_s
+        log_output.should contain("GET")
+        log_output.should match(/#{Time.now.to_s("%Y-%m-%d")}/)
+        called.should be_true
+      ensure
```

Comment:
I'm wondering - isn't context reset between each spec so that the timestamp configured here doesn't leak into other specs? Or it's guarding against some other edge case?

Reply:
Well, I've implemented as per @paulcsmith's description in #210.
Starting with crystal myself so not really sure about details, 
but it's implemented same as ErrorHandler

Reply:
@ugisozols Good question. This is making sure that if an unexpected error happens the configuration is reset. That way it won't break other tests that rely on the timestamp not being shown